packages/light-client/src/utils/logger.ts

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type LogHandler = (message: string, context?: any, error?: Error) => void;
+
+export type ILcLogger = {
+  error: LogHandler;
+  warn: LogHandler;
+  info: LogHandler;
+  debug: LogHandler;
+};
+
+/* eslint-disable no-console */
+
+/**
+ * With `console` module and ignoring debug logs
+ */
+export function getLcLoggerConsole(opts?: {logDebug?: boolean}): ILcLogger {
+  return {
+    error: console.log,
```

Comment:
we could use `console.error` and `console.warn` (and `console.info` and `console.debug`)